src/idris/commands.js

```diff
@@ -247,11 +249,10 @@ let evalSelection = (uri) => {
 }
 
 let startup = (uri) => {
-  term = vscode.window.createTerminal("Idris REPL")
-  let pkgOpts = ipkg.getPkgOpts(innerCompilerOptions)
-  let pkgOptsStr = pkgOpts.length == 0 ? "" : " " + pkgOpts.join(" ")
-  
-  term.sendText(`idris${pkgOptsStr}`)
+  const pathToIdris = vscode.workspace.getConfiguration('idris').get('executablePath') || 'idris';
```

Comment:
Hi @be5invis, I think we can kindly remove `|| 'idris'`? Cause `executablePath` already got its default value as `'idris'`, thanks 😄